package/src/components/MessageInput/ShowThreadMessageInChannelButton.tsx

```diff
@@ -158,17 +129,16 @@ export type ShowThreadMessageInChannelButtonProps =
   Partial<ShowThreadMessageInChannelButtonWithContextProps>;
 
 export const ShowThreadMessageInChannelButton = (props: ShowThreadMessageInChannelButtonProps) => {
+  const { threadList } = useChannelContext();
```

Comment:
Move this back up please. `useChannelContext` cannot be used liberally in leaf components as it contains a bunch more stuff as well. Although the memo guards here, we're still using and consuming the `ChannelContext` in the upstream `MessageInput` so now we're doing stuff twice. 

These types of things introduce performance issues over time.